src/index.js

```diff
@@ -0,0 +1 @@
+import './setup.js';
```

Comment:
- [ ] Your wireframe does not match the specified wireframe because the style file has not yet been imported. Please import the `main.css` file into the `index.js` file as follows: `import './styles/main.css'`.

- Your design:
<img width="347" alt="image" src="https://user-images.githubusercontent.com/77190203/173589745-8ee5c471-a9e1-48d4-875d-56150afe51a5.png">